packages/cli/src/cms/api/api-client.ts

```diff
@@ -70,6 +74,10 @@ class RemotesApiClient {
       }
     );
 
+    if (!response) {
+      throw new Error(`Failed to upsert: Request timed out.`);
```

Comment:
if the request fails due to some other error, we still receive a message Request timeout

Reply:
After discussion, we decided to change this message to more generic one.